tests/integration/plugins/test_cmake.py

```diff
@@ -0,0 +1,91 @@
+# -*- Mode:Python; indent-tabs-mode:nil; tab-width:4 -*-
+#
+# Copyright 2022 Canonical Ltd.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License version 3 as
+# published by the Free Software Foundation.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+import subprocess
+import textwrap
+from pathlib import Path
+
+import pytest
+import yaml
+
+from craft_parts import LifecycleManager, Step
+
+
+@pytest.fixture
+def cmake():
+    subprocess.run(["pip", "install", "cmake"], check=True)
+    yield
+    subprocess.run(["pip", "uninstall", "cmake", "--yes"], check=True)
```

Comment:
This is an interesting way to install cmake, but I think it's better to just install it from the archive when preparing the CI environment in the [github workflow](https://github.com/canonical/craft-parts/blob/7aa01bd45fb8652a09484f46ee77278f03a50f50/.github/workflows/tests.yaml#L82).